pkg/apis/events/v1beta1/conversion.go

```diff
@@ -56,3 +64,31 @@ func Convert_core_Event_To_v1beta1_Event(in *k8s_api.Event, out *v1beta1.Event,
 	out.DeprecatedCount = in.Count
 	return nil
 }
+
+func AddFieldLabelConversionsForEvent(scheme *runtime.Scheme) error {
+	return scheme.AddFieldLabelConversionFunc(SchemeGroupVersion.WithKind("Event"),
+		func(label, value string) (string, string, error) {
+			switch label {
+			case "regarding.kind":
+				return "involvedObject.kind", value, nil
+			case "regarding.namespace":
+				return "involvedObject.namespace", value, nil
+			case "regarding.name":
+				return "involvedObject.name", value, nil
+			case "regarding.uid":
+				return "involvedObject.uid", value, nil
+			case "regarding.apiVersion":
+				return "involvedObject.apiVersion", value, nil
+			case "regarding.resourceVersion":
+				return "involvedObject.resourceVersion", value, nil
+			case "regarding.fieldPath":
+				return "involvedObject.fieldPath", value, nil
+			case "reportingController":
```

Comment:
reportingController is not supported in core/v1:
https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/event/strategy.go#L102

@liggitt  - shouldn't this be changed to `source`?

Reply:
probably so? I'm less familiar with how the v1 fields map to the v1beta1 fields. This does demonstrate the need for a test covering each field

Reply:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/events/event_recorder.go#L88 I added to be able to convert back as this was making some of tests fail during `kube-scheduler`'s migration